docs/README.md

```diff
@@ -38,6 +38,14 @@ The `trustpilot` app also enables Truspilot's widget component to be added to yo
 6. Save your changes;
 7. Access your store's website using the Developer workspace to check out the new component. If no more changes are desired, repeat the steps above in a [Production workspace](https://vtex.io/docs/recipes/development/creating-a-production-workspace/) and then [promote it to Master](https://vtex.io/docs/recipes/development/promoting-a-workspace-to-master).
 
+### Optional
+In case that recibe error on toolbet `I couldn't find an interface "shop-review-badge.trustpilot-widget"` after finish all steps to install, you can add the Trustpilot app to your theme's manifest.json file inside peerDependencies as shown below:
+`````diff
+ "peerDependencies": {
++   "vtex.trustpilot": "1.x"
+ }
+`````
```

Comment:
There shouldn't be that many `s, 3 are already enough :)

Reply:
fixed